caluma/caluma_workflow/domain_logic.py

```diff
@@ -122,15 +122,17 @@ def pre_complete(work_item, validated_data, user, context=None):
         return validated_data
 
     @staticmethod
-    def post_complete(work_item, user, context=None):
+    def post_complete(
+        work_item,
+        user,
+        context=None,
+        event=events.completed_work_item,
```

Comment:
Is there a specific reason to make the event an argument? IMO this should be always the same event

Reply:
Nevermind